### PR TITLE
fix: pub-sub worker issues on cloud

### PR DESCRIPTION
**Pull request by lucasmarshall**

### CODE REVIEW

File: apps/salesforce-pub-sub/index.ts
Comment (lucasmarshall) [inline apps/salesforce-pub-sub/index.ts:1]: Adding awaits fixes error logging stack traces

**Review by albertyfwu — APPROVED** [review]

File: apps/salesforce-pub-sub/index.ts [hunk 1]
```diff
@@ -102,19 +103,35 @@ const { connectionService, integrationService, webhookService, applicationServic
       }
     };
 
-    const subscribe = (eventType: string) => {
-      const streamErrorHandler = (err: any) => {
+    const subscribe = async (eventType: string) => {
```
Comment (tomkit) [inline apps/salesforce-pub-sub/index.ts:106]: What's diff between local and cloud for this?

**Review by tomkit — APPROVED** [review]

File: apps/salesforce-pub-sub/index.ts (hunk 1 above)
Comment (lucasmarshall) [reply apps/salesforce-pub-sub/index.ts:106]: See my comment above

Comment (lucasmarshall) [reply apps/salesforce-pub-sub/index.ts:106]: Making this async fixes some stack trace logging issues

Comment (lucasmarshall) [reply apps/salesforce-pub-sub/index.ts:106]: The error handling were errors I only saw on cloud (we have way more varied connections on cloud that in local testing)

**Pull request merged**